lib/src/rules/unnecessary_overrides.dart

```diff
@@ -54,12 +62,12 @@ class UnnecessaryOverrides extends LintRule {
 
 abstract class _AbstractUnnecessaryOverrideVisitor extends SimpleAstVisitor {
   LintRule rule;
-  Element inheritedMethod;
+  ExecutableElement inheritedMethod;
```

Comment:
Consider making `inheritedMethod` and `declaration` be local variables in `visitMethodDeclaration` and parameters to `_haveSameDeclaration`.